src/utils/hd-wallet.ts

```diff
@@ -23,27 +27,27 @@ export function derivePathFromKey (path, key) {
   return segments.reduce((parentKey, segment) => deriveChild(parentKey, segment), key)
 }
 
-export function derivePathFromSeed (path, seed) {
+export function derivePathFromSeed (path: string, seed: string|Buffer) {
   if (!['m', 'm/'].includes(path.slice(0, 2))) {
     throw new InvalidDerivationPathError()
   }
   const masterKey = getMasterKeyFromSeed(seed)
   return derivePathFromKey(path.slice(2), masterKey)
 }
 
-function formatAccount (keys) {
+function formatAccount (keys: {secretKey: Uint8Array, publicKey: Uint8Array}) {
   const { secretKey, publicKey } = keys
   return {
     secretKey: toHex(secretKey),
     publicKey: encode(publicKey, 'ak')
   }
 }
 
-export function getKeyPair (secretKey) {
-  return nacl.sign.keyPair.fromSeed(secretKey)
+export function getKeyPair (secretKey: Uint8Array|string|Buffer) {
+  return nacl.sign.keyPair.fromSeed(<Uint8Array>secretKey)
```

Comment:
```suggestion
  return nacl.sign.keyPair.fromSeed(secretKey as Uint8Array)
```

Type Assertion using `as` is the recommended way, read more: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-1-6.html#new-tsx-file-extension-and-as-operator